addons/website_slides/models/slide_slide.py

```diff
@@ -562,3 +587,56 @@ def _default_website_meta(self):
         res['default_opengraph']['og:description'] = res['default_twitter']['twitter:description'] = self.description
         res['default_opengraph']['og:image'] = res['default_twitter']['twitter:image'] = "/web/image/slide.slide/%s/image_thumb" % (self.id)
         return res
+
+    def get_user_relation(self, user):
+        relation = self.user_relation_ids.search([('user_id', '=', user)])
+        if relation:
+            return relation.state
+
+    @api.model
+    def reorder_slides(self, slides_data=None):
+        ids = [e['id'] for e in slides_data]
+        slides_to_change = self.browse(ids)
+        for slide in slides_data:
+            s = slides_to_change.browse(slide['id'])
```

Comment:
Public method that sudo writes fields.
That means that anyone can reorder other people's slides.

Should be checked I think.

Reply:
Done. Replaced that method by a controller